pkg/controller/gitopscluster/gitopscluster_controller.go

```diff
@@ -1165,7 +1167,8 @@ func getManagedClusterURL(managedCluster *spokeclusterv1.ManagedCluster, token s
 		httpClient := http.DefaultClient
 		httpClient.Transport = &http.Transport{
 			TLSClientConfig: &tls.Config{
-				RootCAs: caCertPool,
+				RootCAs:    caCertPool,
+				MinVersion: tls.VersionTLS12,
```

Comment:
We may use tls.VersionTLS13. I remember it has been ready and it is suggested by security squad.